### ISSUE TITLE
Out of memory when running on TIKE

### ISSUE THREAD

**Issue by mrtommyb** [tessgi/tesscube#8]
I run the following code on TIKE and my memory quickly blows up over 15 GB.

```python
from tesscube import TESSCube
ticid, ra, dec, sector, camera, ccd, row, col = (220433364, 73.377272, -55.858796, 31, 3, 4, 242.54, 109.16)
r = TESSCube(sector=sector, camera=camera, ccd=ccd)
corner = (row, col)
tpf = r.get_tpf(corner, shape=(15,15))
```

**Comment by mrtommyb** [comment]
I think the issue is likely that I'm passing in floats and it requires integers. In which case I can make a PR.

**Comment by christinahedges** [comment]
Thanks @mrtommyb! This is now addressed in v1.0.1dev
